releng/sap-cf-base/src/main/java/org/eclipse/dirigible/cf/utils/CloudFoundryUtils.java

```diff
@@ -85,6 +85,12 @@ public static DestinationEnv getDestinationEnv() {
 		return vcapServicesEnv.getDestinationEnv() != null ? vcapServicesEnv.getDestinationEnv().get(0) : null;
 	}
 
+	public static ConnectivityEnv getConnectivityEnv() {
+		String envJson = EnvFacade.get(VCAP_SERVICES);
+		VcapServicesEnv vcapServicesEnv = GsonHelper.GSON.fromJson(envJson, VcapServicesEnv.class);
+		return vcapServicesEnv.getDestinationEnv() != null ? vcapServicesEnv.getConnectivityEnv().get(0) : null;
```

Comment:
Shouldn't this be:

```java
return vcapServicesEnv.getConnectivityEnv() != null ? vcapServicesEnv.getConnectivityEnv().get(0) : null;
```

So instead of `vcapServicesEnv.getDestinationEnv()` -> `vcapServicesEnv.getConnectivityEnv()`